docs/source/noise/index.rst

```diff
@@ -165,10 +165,10 @@ Noise types for each column
     - W-2 and 1099, SSA
     - "Borrowed" SSN, leave a field blank, write the wrong digits, make typos
     - Note that "borrowed" SSN only applies to the W-2 and 1099 dataset
-  * - Income / wages
+  * - Income 
     - W-2 and 1099
     - Leave a field blank, make typos
```

Comment:
I just noticed that "write the wrong digits" is missing from the possible noise types for income. Let's go ahead and add that here so we don't have to make a whole nother hot-fix for it.

Reply:
Done